lib/puppet/ssl/oids.rb

```diff
@@ -61,6 +61,7 @@ module Puppet::SSL::Oids
     ["1.3.6.1.4.1.34380.1.1.23", 'pp_cloudplatform', 'Puppet Node Cloud Platform Name'],
     ["1.3.6.1.4.1.34380.1.1.24", 'pp_apptier', 'Puppet Node Application Tier'],
     ["1.3.6.1.4.1.34380.1.1.25", 'pp_hostname', 'Puppet Node Hostname'],
+    ["1.3.6.1.4.1.34380.1.1.26", 'pp_owner', 'Puppet Node Owner'],
```

Comment:
I think a similar change will be needed in https://github.com/puppetlabs/puppetserver-ca-cli/blob/a18b7a2b2e5632d2657905406c6c6df5e29b90fc/lib/puppetserver/ca/host.rb#L49 (heads up @Magisus)

Reply:
Submitted https://github.com/puppetlabs/puppetserver-ca-cli/pull/58